nav2_theta_star_planner/src/theta_star_planner.cpp

```diff
@@ -154,13 +153,13 @@ void ThetaStarPlanner::getPlan(nav_msgs::msg::Path & global_path)
 {
   std::vector<coordsW> path;
   if (planner_->isUnsafeToPlan()) {
-    RCLCPP_ERROR(logger_, "Either of the start or goal pose are an obstacle! ");
     global_path.poses.clear();
+    throw nav2_core::PlannerException("Either of the start or goal pose are an obstacle! ");
```

Comment:
Check test coverage - exceptions are being thrown in unit tests that need to be caught so they continue for error condition testing.

Reply:
If there are some checks missing, maybe also worth adding them